OpenVPNHelper/OpenVPNHelper-Info.plist

```diff
@@ -19,7 +19,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>0.2</string>
 	<key>CFBundleVersion</key>
-	<string>20</string>
+	<string>21</string>
```

Comment:
If you increase this version, you also need to change there version in the related Service used for the version check.